internal/processor/filterspan/config.go

```diff
@@ -100,3 +104,16 @@ type Attribute struct {
 	// If it is not set, any value will match.
 	Value interface{} `mapstructure:"value"`
 }
+
+type InstrumentationLibrary struct {
```

Comment:
Comment if exported. Does this need to be exported?

Reply:
actually not - neither does Attribute. Why did you export `Attribute`? 
It's kind of a public interface, as it has the json structure.